native/utils.h

```diff
@@ -19,4 +20,16 @@ void cv_to_ffi(const std::vector<T>& source, CVec<U>* dest) {
         cv_to_ffi(source[i], &dest->array[i]);
     }
 }
+
+void ffi_to_cv(const cv::Mat& source, cv::Mat* dest);
+
+template <typename T, typename U>
+void ffi_to_cv(const CVec<U*>& source, std::vector<T>* dest) {
+    dest->reserve(source.size);
+    for (size_t i = 0; i < source.size; i++) {
+        T* cell = new T();
```

Comment:
It's probably a memory leak. Do we need `new` here? I was affraid that when T goes out of scope we can reference freed memory.

Reply:
I think create an object `T` directly should be OK as it's pushed into the `vector` (i.e. ownership being managed by `vector`).